src/sources/metadatasourcetaglib.cpp

```diff
@@ -313,8 +313,7 @@ class MpegTagSaver : public TagSaver {
             : m_file(TAGLIB_FILENAME_FROM_QSTRING(fileName)),
               m_modifiedTagsBitmask(exportTrackMetadata(&m_file, trackMetadata)) {
     }
-    ~MpegTagSaver() override {
-    }
+    ~MpegTagSaver() = default;
```

Comment:
The override keyword is lost. Please check other occurrences as well.

Reply:
Good catch :see_no_evil: